docs/source/conf.py

```diff
@@ -299,6 +299,8 @@
     "node_arg_is_bias",
     "node_arg_is_weight",
     "return_arg_list",
+    # torch.ao.quantization.pt2e.export_utils
+    "model_is_exported",
```

Comment:
Adding new entries in this file is not ok.
Please properly document this API so that it appears properly on the website. Or make it private